api/segments/migrations/0023_add_versioning_to_segments.py

```diff
@@ -5,6 +5,16 @@
 from django.db import migrations, models
 
 
+def _get_sql_file_content(filename: str) -> str:
+    return open(
+        os.path.join(
+            os.path.dirname(__file__),
+            "sql",
+            filename,
+        )
+    ).read()
```

Comment:
Please be consistent with [other migrations that do this](https://github.com/Flagsmith/flagsmith/blob/36e634ca057e6aa55ffed41686a05df0883a1062/api/features/versioning/managers.py#L12-L13).

Reply:
This is done.